tests/test_template_generation.py

```diff
@@ -0,0 +1,47 @@
+import json
+import tempfile
+from pathlib import Path
+from unittest import TestCase
+
+from samcli.commands.pipeline.init.interactive_init_flow import (
+    _initialize_pipeline_template,
+)
+
+
+class TestTemplateGeneration(TestCase):
```

Comment:
Can you add instructions on how to run this? Either in README or in python doc here.

Reply:
added